src/js/components/shepherd-modal.svelte

```diff
@@ -231,6 +231,7 @@
     height: 100vh;
     opacity: 0.5;
     transition: all 0.3s ease-out, height 0s 0s, opacity 0.3s 0s;
+    -webkit-transform: translateZ(0);
```

Comment:
```suggestion
    transform: translateZ(0);
```

@monshan I wonder if we can use just normal `transform` here? Not sure we need the `webkit` prefix anymore. Did you test this fix and it fixed the issue?

Reply:
@rwwagner90 yes when I did the dev build with this webkit prefix it worked for me, just tried the normal transform though and it also works. I'll re-commit shortly 👍